devel/nixpacks/Portfile

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        railwayapp nixpacks v0.3.9
+github.tarball_from archive
+
+name                nixpacks
+version             0.3.9
```

Comment:
the "name" and "version" fields are set automatically by the github PG and thus can/should be removed.